docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -35,4 +35,23 @@
     
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Dynamic\Trainers\Regression\FastTreeTemplate.tt">
```

Comment:
>FastTreeTemplate.tt" [](start = 46, length = 20)

is this Runtime Text Template or Text Template? At what point does it run?

Reply:
To be more specific, it is design (or compile) time template. In VS studio,  `FastTreeTemplate.cs` is generated from this file as soon as the .tt file is saved (Ctrl + S) in the VS.

---
In reply to: [267099394](https://github.com/dotnet/machinelearning/pull/3017#discussion_r267099394) [](ancestors = 267099394)